### ISSUE TITLE
Add directories to working sets

### ISSUE THREAD

**Issue by JustinEWalker** [bernardop/vscode-working-sets#4]
I'm working in a very large project with literally tens of thousands of directories. I will only be working in a couple of directories at one time but expanding the directory tree to navigate to the few directories I'm interested is a) time consuming and b) results in a lot of scrolling in the explorer in order to jump between the directories I'm interested in.

I'd love the ability in vscode-working-sets to right-click a directory in explorer and add it and all of it's files and sub-directories to a working set. That way I could see only the directories and files that I'm actually interested in for the particular issue I'm working on.

**Comment by bernardop** [comment]
Hey @JustinEWalker are you thinking something like what Nuclide offers? https://nuclide.io/docs/features/working-sets/

In my mind, the philosophy for this project was to have temporary and not-too-large working sets. Something to help with context switching when working in different tasks. I've seen some discussions in the vscode repo that might tackle a solution for your use-case in the future.

- microsoft/vscode#9498
- microsoft/vscode#87666

**Comment by JustinEWalker** [comment]
@bernardop Yes, it looks like that feature in Nuclide is what I was thinking. The discussion in https://github.com/microsoft/vscode/pull/87666 looks very promising but seems to have stalled in February.

What I'm thinking of is not too far off from your vision. I am working in one or two subdirectories that contain around 50 files. Currently I can achieve what I want with your extension, but I have to manually add every one of the 50 files that I might be interested in. I'd love to add the entire subdirectory to save me the hassle of adding them one-at-a-time.

**Comment by bernardop** [comment]
Ah, that sounds perfectly doable. I can take a crack at it in the next couple of weeks.
